__tests__/app.test.js

```diff
@@ -293,4 +293,97 @@ describe("app", () => {
         });
     });
   });
+  describe("/api/articles/:article_id/comments - POST", () => {
+    test("status 200 - adds a comment to the comment table", () => {
+      const newComment = {
+        username: "lurker",
+        body: "first ever review"
+      };
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(200)
+        .then(() => {
+          return request(app).get("/api/articles/2/comments");
+        })
+        .then(({ body: { comments } }) => {
+          expect(comments[0].body).toBe("first ever review");
+        });
+    });
```

Comment:
should just return comments as an object instead of an array( return rows[0] in your model instead of rows)